mllib/src/main/scala/org/apache/spark/mllib/clustering/DistanceMeasure.scala

```diff
@@ -100,6 +100,18 @@ private[spark] abstract class DistanceMeasure extends Serializable {
     new VectorWithNorm(sum)
   }
 
+  /**
```

Comment:
Is the above `def centroid(sum: Vector, count: Long): VectorWithNorm` still needed?

Reply:
Yes. It is still used by ```BisecttingKMeans```